src/wazuh_db/wdb.h

```diff
@@ -1873,13 +1864,23 @@ cJSON* wdb_global_select_group_belong(wdb_t *wdb, int id_agent);
 int wdb_global_insert_agent_belong(wdb_t *wdb, int id_group, int id_agent, int priority);
 
 /**
- * @brief Function to delete a group from belongs table using the group name.
+ * @brief Function to remove an agent from the belongs table.
```

Comment:
We may update this description. The method only removes one entry of in the `belongs` table.

Reply:
Fixed